### ISSUE TITLE
java.lang.RuntimeException: Unable to create application org.electroncash.electroncash3.App: com.chaquo.python.PyException: AttributeError: module 'electroncash.locale' has no attribute '__file__'

### ISSUE THREAD

**Issue by ProfessorVenkman** [Electron-Cash/Electron-Cash#1562]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5743)
	at android.app.ActivityThread.-wrap1(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1656)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6494)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
Caused by: com.chaquo.python.PyException: AttributeError: module 'electroncash.locale' has no attribute '__file__'
	at <python>.<module>(/android_asset/chaquopy/app.zip/electroncash/i18n.py:35)
	at <python>.load_module_impl(/data/user/0/org.electroncash.wallet/files/chaquopy/bootstrap.zip/java/android/importer.py:435)
	at <python>.load_module(/data/user/0/org.electroncash.wallet/files/chaquopy/bootstrap.zip/java/android/importer.py:353)
	at <python>._load_backward_compatible(<frozen importlib._bootstrap>:626)
	at <python>._load_unlocked(<frozen importlib._bootstrap>:656)
	at <python>._find_and_load_unlocked(<frozen importlib._bootstrap>:955)
	at <python>._find_and_load(<frozen importlib._bootstrap>:971)
	at <python>.java.chaquopy.import_override(import.pxi:18)
	at <python>.<module>(/android_asset/chaquopy/app.zip/electroncash/util.py:39)
	at <python>.load_module_impl(/data/user/0/org.electroncash.wallet/files/chaquopy/bootstrap.zip/java/android/importer.py:435)
	at <python>.load_module(/data/user/0/org.electroncash.wallet/files/chaquopy/bootstrap.zip/java/android/importer.py:353)
	at <python>._load_backward_compatible(<frozen importlib._bootstrap>:626)
	at <python>._load_unlocked(<frozen importlib._bootstrap>:656)
	at <python>._find_and_load_unlocked(<frozen importlib._bootstrap>:955)
	at <python>._find_and_load(<frozen importlib._bootstrap>:971)
	at <python>.java.chaquopy.import_override(import.pxi:18)
	at <python>.<module>(/android_asset/chaquopy/app.zip/electroncash/__init__.py:2)
	at <python>.load_module_impl(/data/user/0/org.electroncash.wallet/files/chaquopy/bootstrap.zip/java/android/importer.py:435)
	at <python>.load_module(/data/user/0/org.electroncash.wallet/files/chaquopy/bootstrap.zip/java/android/importer.py:353)
	at <python>._load_backward_compatible(<frozen importlib._bootstrap>:626)
	at <python>._load_unlocked(<frozen importlib._bootstrap>:656)
	at <python>._find_and_load_unlocked(<frozen importlib._bootstrap>:955)
	at <python>._find_and_load(<frozen importlib._bootstrap>:971)
	at <python>._gcd_import(<frozen importlib._bootstrap>:994)
	at <python>._call_with_frames_removed(<frozen importlib._bootstrap>:219)
	at <python>._find_and_load_unlocked(<frozen importlib._bootstrap>:941)
	at <python>._find_and_load(<frozen importlib._bootstrap>:971)
	at <python>._gcd_import(<frozen importlib._bootstrap>:994)
	at <python>.import_module(stdlib/importlib/__init__.py:126)
	at <python>.chaquopy_java.Java_com_chaquo_python_Python_getModule(chaquopy_java.pyx:153)
	at com.chaquo.python.Python.getModule(Native Method)
	at org.electroncash.electroncash3.DaemonKt.libMod(Daemon.kt:15)
	at org.electroncash.electroncash3.NetworkKt$libNetwork$2.invoke(Network.kt:30)
	at org.electroncash.electroncash3.NetworkKt$libNetwork$2.invoke(Unknown Source:0)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at org.electroncash.electroncash3.NetworkKt.getLibNetwork(Unknown Source:7)
	at org.electroncash.electroncash3.SettingsKt.setDefaultValues(Settings.kt:38)
	at org.electroncash.electroncash3.SettingsKt.initSettings(Settings.kt:23)
	at org.electroncash.electroncash3.App.onCreate(App.kt:47)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1119)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5740)
	... 8 more
java.lang.RuntimeException: Unable to create application org.electroncash.electroncash3.App: com.chaquo.python.PyException: AttributeError: module 'electroncash.locale' has no attribute '__file__'
```

Reporter
------------

This issue was reported by 1 user(s):

| Electron Cash Version  | Python Version | Operating System  | Wallet Type  | Locale |
|---|---|---|---|---|
| 4.0.8-0  | 3.6.5 (default, Dec  7 2018, 14:06:05)  [GCC 4.9] | Android 8.1.0 on Google Android SDK built for x86 (generic_x86) | unknown | en_US | 

Additional Information
------------------------

The reporting user(s) did not provide additional information.

**Comment by cculianu** [comment]
I hope.. this is not in production.

@mhsmith: Should have used `__path__` man.. should have used `__path__`...  ;)

**Comment by mhsmith** [comment]
I never released a 4.0.8, only 4.0.7 pre-release builds and 4.0.9 currently on Google Play. This appears to be someone making their own build of something in between (possibly with their own modifications) and running it on an emulator.

Unless it happens with the production build, I don't think any action is required.

**Comment by cculianu** [comment]
Gotcha. Closing.
